### PR TITLE
Include app URL to the project

### PR DESCRIPTION
**Pull request by serhii73**

### CODE REVIEW

**Comment by serhii73** [comment]
@webknjaz All good? Can we merge this branch to master?

**Comment by webknjaz** [comment]
Rebase it and get rid of unrelated commits.

**Comment by serhii73** [comment]
@webknjaz Done.

File: blog/urls.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""URLs for the blog app."""
+from django.urls import re_path
+from . import views
+
+
+urlpatterns = [
+    re_path(r'^$', views.post_list, name='post_list'),
```
Comment (webknjaz) [inline blog/urls.py]: This object does not exist yet in the current commit. It's bad to have "broken" commits in repo. You should aim to have "working" project in all commits of the repo.

Comment (serhii73) [reply blog/urls.py]: Sorry, @webknjaz  But I thought 1 page from tutorial it one branch and PR here.

Comment (webknjaz) [reply blog/urls.py]: Okay, then it should be merged after the other PR with the mentioned view function available.

Comment (webknjaz) [reply blog/urls.py]: Could you please also add another PR integrating pyflakes into the pre-commit tool first?

**Comment by serhii73** [comment]
@webknjaz Can we to merge the PR?

**Comment by serhii73** [comment]
@webknjaz Maybe close this PR and to merge just https://github.com/kpi-web-guild/django-girls-blog-serhii73/pull/25 ?

**Comment by serhii73** [comment]
@webknjaz But I can't merge the PR in master, because Travis is red.
`blog/urls.py:L   7:(no-member) Module 'blog.views' has no 'post_list' member (C 19), ::E1101`
I think I need to unite https://github.com/kpi-web-guild/django-girls-blog-serhii73/pull/24 with https://github.com/kpi-web-guild/django-girls-blog-serhii73/pull/25 .
For example, if I here I need to run `git merge feature/6/django_views` or to do the opposite?

**Comment by webknjaz** [comment]
That's the whole point to having Travis.

**Comment by webknjaz** [comment]
The approval means that I agree with what I see in PR, it doesn't mean that it is safe to merge. And review is about high-level things, which are not checked by robots.

**Comment by serhii73** [comment]
@webknjaz What do you recommend to do?

**Comment by serhii73** [comment]
@webknjaz What do you recommend to do?

**Comment by webknjaz** [comment]
Merge #25

**Comment by webknjaz** [comment]
(to master!)

**Comment by serhii73** [comment]
@webknjaz done

**Pull request merged**